app/lib/todos.server.ts

```diff
@@ -3,6 +3,7 @@ export interface Todo {
   text: string;
   completed: boolean;
   createdAt: Date;
+  status?: string;
```

Comment:
suggestion: use KanbanStatus for strict types
```suggestion
  status?: KanbanStatus;
```
Note: import type from schema file.